src/Test/Operator/Elastic.H

```diff
@@ -86,13 +90,50 @@ public:
 	///     \f[\alpha u_i^{mn} \pi^2 (m^2 + n^2) = b_i^{mn}\f]
 	/// or, simplifying
 	///     \f[u_i^{mn}  = \frac{b_i^{mn}}{(\alpha\ \pi^2\ (m^2 + n^2))}\f]
-	/// 
 	///
 	int TrigTest(int verbose,
 		     int component = 0,
 		     int n = 1,
 		     std::string plotfile = "");
-	int mytest() {return 8;};
+
+        /// Compute the exact solution of the governing equation
+        ///   \f[ \sigma_{ij} = C_{ijkl} \varepsilon_{kl}  \f]
+        /// Where
+        ///   \f[C_{ijkl} = \mu (\delta_{ik}\delta_{jl + \delta_{il} \delta_{jk}}) \lambda \delta_{ij}\delta_{kl} \f]
+        /// Stress-strain relationship
+        ///   \f[ \sigma_{ij} = 2\mu\varepsilon_{ij} + \lambda\varepsilon_{kk}\delta_{ij}  \f]  
+	/// (see Model::Solid::LinearElastic::Isotropic for implementation of this model)
+  	/// In this case the two Lamé parameters are:
+	///   \f[ \mu=2.6    \lambda=6.0 \f]
+        ///
+	/// Consider a simulation domain \f$\Omega = [0, 1]^3\f$
+        /// And a subjected material to stress in one direction but constrained in the other directions
+        /// so that the material only can contract or expand on one side,
+        /// i.e it will deformate only on the testing component side (Uniaxial strain).
+        ///
+        /// Component corresponds to each one 3D directions
+        ///   \f[i=0; X direction  -  i=1; Y direction  -  i=2; Z direction \f]
+        /// 
+	/// Expand \f$\mathbf{u}\f$ and \f$\mathbf{b}\f$ out as affine series
```

Comment:
This statement is not correct. There is no such thing as an "affine series" (that I'm aware of.) 
It would be more accurate to say "Let the solution *u* and right hand side *b* have the form"